src/app/space/create/deployments/services/deployments.service.spec.ts

```diff
@@ -147,126 +153,233 @@ describe('DeploymentsService', () => {
   });
 
   describe('#isDeployedInEnvironment', () => {
-    it('should be true', fakeAsync(() => {
-      svc.isDeployedInEnvironment('foo', 'stage')
-        .subscribe((active: boolean) => {
-          expect(active).toBeTruthy();
-        });
-      tick(DeploymentsService.POLL_RATE_MS + 10);
-      discardPeriodicTasks();
-    }));
+    it('should be true for included environments', (done: DoneFn) => {
+      const expectedResponse = {
+        data: {
+          applications: [
+            {
+              name: 'vertx-hello',
+              pipeline: [
+                {
+                  name: 'run'
+                }
+              ]
+            }
+          ]
+        }
+      };
```

Comment:
On second thought, could we get a few more test cases against empty and multiple spaces/applications/environments?